docs/sources/alerting/alerting-rules/alerting-migration/migration-api.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Use the Grafana Alerting API import tool to convert your datasource managed alert rules into Grafana managed alert rules
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Import data source-managed alert rules with Grafana Mimirtool
+menuTitle: API alert rules import
+weight: 601
+refs:
+---
+
+# Import data source-managed alert rules with Grafana Mimirtool
+
+You can convert data source-managed alert rules to Grafana-managed alert rules with the Grafana import tool in the Grafana user interface, or you can convert them with the Grafana Mimirtool command-line tool. This guide tells you how to use Mimirtool to import your data source-managed alert rules.
+
+## Before you begin
+
+To import data source-managed alert rules with Grafana Mimirtool, you need to have the Grafana Mimirtool command-line tool installed.
+
+You need a service account the following [RBAC permissions](/docs/grafana/latest/administration/roles-and-permissions/access-control/):
+
+- Alerting: Rules Reader
+- Alerting: Rules Writer
+- Alerting: Set provisioning status
+- Datasources: Reader
+- Folders: Creator
+- Folders: Reader
+- Folders: Writer
+
+You also need to create a service account token with your service account. Refer to the [documentation for more information on service accounts and service account tokens](/docs/grafana/latest/administration/service-accounts/)
+
+## How it works
+
+When data source-managed alert rules are converted to Grafana-managed alert rules, the following are applied to the Grafana-managed alert rules:
+
+- All rules are given `rule_query_offset` offset value of 1m.
```

Comment:
Maybe we should mention that this is configurable in [the config](https://github.com/grafana/grafana/blob/c83c486c9b3a123f799073a548c39760492e0926/conf/sample.ini#L1508) if users are using OSS or Enterprise:

```
[unified_alerting.prometheus_conversion]
rule_query_offset = 1m
```

By default it's 1m and will be the same in the Cloud. But if the `query_offset` is explicitly configured in the rule group itself ([Prometheus docs](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#rule_group)), that value takes precedence over the configuration setting.